index.html

```diff
@@ -349,6 +343,42 @@
     </section>
 
     <section>
+      <h2>Extensions to the <a>DocumentOrShadowRoot</a> Mixin</h2>
+
+      <dl title='partial interface DocumentOrShadowRoot' class='idl'>
+         <dt>readonly attribute Element? pointerLockElement</dt>
+         <dd>
+           <dfn title="pointerLockElement"></dfn>
+
+           <p>If the element, which is set as the target for mouse events while the pointer is
+           locked, is a
+           <a href="https://dom.spec.whatwg.org/#concept-shadow-including-descendant">shadow-including descendant</a> of the
+           <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+           return the result of the following algorithm:
+           <ol>
+             <li>Let <var>target</var> be the target for mouse events while the pointer is locked.
+             <li>If <var>target</var> is in the same tree as the
+               <a href="http://dom.spec.whatwg.org/#context-object">context object</a>,
+               let <var>adjusted-target</var> be <var>target</var></li>
+             <li>Otherwise: let <var>adjusted-target</var> be the shadow host of <var>target</var> and repeat the following substeps:
```

Comment:
Updated to use "A's root's host" instead of "A's shadow host".

The precondition requires "the target is in the shadow-including descendant of the context object", i.e. if the context's root is not the same as the target's, the target's root must be a shadow root.